src/quicreach.cpp

```diff
@@ -208,15 +209,19 @@ bool TestReachability(const ReachConfig& Config) {
             auto InitialTime = (uint32_t)(Connection.Stats.TimingInitialFlightEnd - Connection.Stats.TimingStart);
             auto Amplification = (double)Connection.Stats.RecvTotalBytes / (double)Connection.Stats.SendTotalBytes;
             auto TooMuch = false, MultiRtt = false;
+            auto Retry = (bool)(Connection.Stats.StatelessRetry);
             if (Connection.Stats.SendTotalPackets != 1) {
                 MultiRtt = true;
                 ++Results.MultiRttCount;
             } else {
                 TooMuch = Amplification > 3.0;
                 if (TooMuch) ++Results.TooMuchCount;
             }
+            if (Retry) {
+              ++Results.RetryCount;
+            }
             if (Config.PrintStatistics)
-                printf("    %3u.%03u ms    %3u.%03u ms    %3u.%03u ms    %u:%u %u:%u (%2.1fx)    %4u    %4u    %s     %c",
+                printf("    %3u.%03u ms    %3u.%03u ms    %3u.%03u ms    %u:%u %u:%u (%2.1fx)    %4u    %4u    %s     %c     %c",
```

Comment:
Instead of having a space between all the trailing characters, I'd rather have them next to each other. Maybe pre-build up the string of characters in a buffer and change this to `%s` and pass that in.